### PR TITLE
Adding options to export cells, materials, temps, densities separately.

### PR DESCRIPTION
**Pull request by pshriwise**
This addresses #35. 

I've also wrapped `ExportTallyDialog.export_data` method to make sure the plot view is reset correctly, even if the export fails for some reason.

### CODE REVIEW

**Comment by pshriwise** [comment]
Very good point. I'll update the menu item to simply `Export...` and allow exports when no tally is selected.

File: tools.py [hunk 1]
```diff
@@ -272,11 +374,6 @@ def export_data(self):
         progressBar.setLabel(QtWidgets.QLabel("Export complete"))
         progressBar.setValue(res[2])
 
-        # restore the previous active
-        self.model.currentView = cv
-        self.model.activeView = av
-        self.model.makePlot()
-
         msg = QtWidgets.QMessageBox()
         msg.setText("Tally Export Complete!")
```
Comment (paulromano) [inline tools.py]:
```suggestion
        msg.setText("Export Complete!")
```

**Comment by paulromano** [comment]
Looking good now; thanks for the changes. One suggestion above but otherwise I think it's good to go.

**Comment by paulromano** [comment]
Thanks @pshriwise!

**Pull request merged**